modules/nf-core/cellpose/main.nf

```diff
@@ -6,7 +6,7 @@ process CELLPOSE {
     if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
         exit 1, "I did not manage to create a cellpose module in Conda that works in all OSes. Please use Docker / Singularity / Podman instead."}
 
-    container "biocontainers/cellpose:2.1.1_cv2"
+    container "docker.io/biocontainers/cellpose:2.1.1_cv2"
```

Comment:
Weird that some biocontainer images are on docker.io and not quay.io!

Reply:
We could create a mulled container with just that tool for the instances missing from quay. This way they will become available as a mulled container, I don't know if you think it is worth the effort. If yes I can take care of creating them.